insights/parsers/libssh_config.py

```diff
@@ -0,0 +1,80 @@
+"""
+Libssh configuration files
+==========================
+
+This module provides the paths of configuration file for libssh package
+
+Parsers provided by this module are:
+
+LibsshClientConfig - file ``/etc/libssh/libssh_client.config``
+--------------------------------------------------------------
+
+LibsshServerConfig - file ``/etc/libssh/libssh_server.config``
+--------------------------------------------------------------
+"""
+
+import re
+from insights.specs import Specs
+from insights.core import CommandParser
+from .. import parser, get_active_lines
+from insights.parsers import SkipException
+
+# regex for matching the 'include' directives
+INCLUDE_RE = re.compile(r'^Include\s+\S+')
+
+
+class LibsshConfig(CommandParser, list):
+    """
+    Base class for libssh configuration file.
+
+    Sample output::
+
+        # Parse system-wide crypto configuration file
```

Comment:
I can't believe these files only contain the `include` lines?

Reply:
I searched doc about on google but found nothing and there is no man page for them.

Reply:
If you are not sure about the content of this file, I can't believe this spec will get approved. You need to get the spec approved at first.